gammapy/data/event_list.py

```diff
@@ -481,19 +481,20 @@ def check(self, checks="all"):
         checker = EventListChecker(self)
         return checker.run(checks=checks)
 
-    def _get_coord_from_geom(self, geom):
-        """Return MapCoord of the EventList relative to an input geometry.
+    def map_coord(self, geom):
+        """Event map coordinates for a given geometry.
 
         Parameters
         ----------
         geom : `~gammapy.maps.Geom`
-            the geom used to define the MapCoord
+            Geometry
 
         Returns
         -------
-        mapcoord : `~gammapy.Map.MapCoord`
+        coord : `~gammapy.maps.MapCoord`
```

Comment:
The method currently returns a `dict` and not a `MapCoord` object. I guess there are two options:
- Rename method to just `coords()` an keep returning a dict
- Change to create a `MapCoord` object inside

Reply:
I'll change to return a `MapCoord` object